backend/de.metas.business/src/main/java/de/metas/invoice/invoiceProcessingServiceCompany/InvoiceProcessingServiceCompanyConfig.java

```diff
@@ -90,10 +91,24 @@ public Optional<Percent> getFeePercentageOfGrandTotalByBpartner(@NonNull final B
 		}
 
 		return detailsList.stream()
+				.filter(InvoiceProcessingServiceCompanyConfigBPartnerDetails::getIsActive)
 				.filter(details -> details.getDocTypeId() == null)
 				.map(InvoiceProcessingServiceCompanyConfigBPartnerDetails::getPercent)
 				.findFirst();
 	}
+
+	public boolean isBPartnerDetailsActive(@NonNull final BPartnerId bpartnerId)
+	{
+		final ImmutableCollection<InvoiceProcessingServiceCompanyConfigBPartnerDetails> detailsList = bpartnerDetails.get(bpartnerId);
+
+		if (isEmpty(detailsList))
```

Comment:
why not simply calling detailsList.isEmpty()?

Reply:
you are right, since ImmutableMultimap does not permit null values, we don't have any reason to use Check.isEmpty, therefore we can directly use detailsList.isEmpty
it was old code, will do the improvement